test/e2e/components/vl-form-validation.js

```diff
@@ -1,6 +1,37 @@
 const { VlElement } = require('vl-ui-core').Test;
 
+
 class VlFormValidation extends VlElement {  
+       
+    async isRequired() {
+        return this.hasAttribute('data-required');
+    }
+
+    async getErrorMessage() {
+        return this.getAttribute('data-vl-error-message');
+    }
+
+    async getErrorPlaceholder() {
+        return this.getAttribute('data-vl-error-placeholder');
+    }
+
+    async getErrorClass() {
+        return this.getAttribute('data-vl-error-class');
+    }
+
+    async getSuccessClass() {
+        return this.getAttribute('data-vl-success-class')
+    }
+
+    async hasError() {
+        const errorClass = await this.getErrorClass();
+        return this.hasClass(errorClass);
+    }
+
+    async isSuccess() {
+        const successClass = await this.getSuccessClass();
+        return this.hasClass(successClass);
+    }
```

Comment:
Kan volgens mij volledig vervangen worden door:
```suggestion
    async hasError() {
        return this.hasClass('data-vl-error-class');
    }

    async isSuccess() {
        return this.hasClass('data-vl-success-class');
    }
```

Reply:
Suggestie klopt toch niet, het is de waarde van het attribuut, niet de attribuut naam.